schemas/BaseElements.xsd

```diff
@@ -517,80 +517,138 @@
 			<xs:element name="AtticAndRoof" minOccurs="0">
 				<xs:complexType>
 					<xs:sequence>
-						<xs:element minOccurs="0" name="Roofs">
+						<xs:element minOccurs="0" name="Attics">
```

Comment:
It feels a little silly to have both AtticAndRoof and Attics within it. Perhaps remove the AtticAndRoof element?